docs/cloud-workload-security/backend.md

```diff
@@ -3369,5 +3369,5 @@ CSM Threats logs have the following JSON schema:
 
 
 
-[1]: /security/threats/
-[2]: /security/threats/agent_expressions
+[1]: https://docs.datadoghq.com/security/threats/
```

Comment:
Those files are directly imported by the docs website, so the relative path actually works there. We would like to keep it as is

Reply:
Oh ok I had no idea my bad, I'll revert that
Unfortunately it means those links are not working when looking from the Github UI...